installability.fmf

```diff
@@ -1,21 +1,38 @@
 summary: TMT/FMF plan for running the installability tests.
 description: Runs installability tests in Fedora CI — https://github.com/fedora-ci/installability-pipeline.
 
-provision:
-    how: virtual
+environment:
+  # Make sure mini-tps can find Koji
+  BREWHUB: https://koji.fedoraproject.org/kojihub
+  BREWROOT: https://kojipkgs.fedoraproject.org
+
+prepare:
+  - name: Install installability executer
+    how: install
+    copr: "@osci/mini-tps"
+    package: mini-tps
+    # TODO: These should be included in the mini-tps package
+  - name: Install mini-tps dependencies
+    how: install
+    package:
+      - getopt
+      - awk
+      - ps
```

Comment:
Do you think these should be added to [mini-tps.spec](https://github.com/fedora-ci/mini-tps/blob/main/mini-tps.spec)?

Reply:
Yes, but I did not double-check if these are the only ones needed. A more detailed review of the bash files in there is necessary